pkg/geodata/ckmeans.go

```diff
@@ -128,15 +130,15 @@ AND geo.valid
 
 -- the category we are interested in
 AND nomis_category.long_nomis_code IN ($2, $3)
-AND nomis_category.year = 2011
+AND nomis_category.year = $4
```

Comment:
as above, should this be `nomis_cateogory.year = data_ver.year` and `data_ver.year = $4`?